### PR TITLE
fix/twap and cond quote token indexing

### PR DESCRIPTION
**Pull request by LukasDeco**
- **fix: twaps indexing missing obs and price**
- **fix: pass and fail usdc logo ternary wrong**

### CODE REVIEW

File: packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts [hunk 1]
```diff
@@ -495,8 +495,8 @@ async function insertAssociatedAccountsDataForProposal(
     if (dao && daoDetails) {
       if (isQuote) {
         // Fail / Pass USDC
-        imageUrl = isFail
-          ? "https://imagedelivery.net/HYEnlujCFMCgj6yA728xIw/f38677ab-8ec6-4706-6606-7d4e0a3cfc00/public"
+        imageUrl = !isFail
```
Comment (R-K-H) [inline packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts:498]: This is backwards now.

**Review by R-K-H — CHANGES_REQUESTED** [review]

File: packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts (hunk 1 above)
Comment (LukasDeco) [reply packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts:498]: that's the point

Comment (LukasDeco) [reply packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts:498]: bro this is the image it was using for fail:
pUSDC

Comment (LukasDeco) [reply packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts:498]: oh shit forget all I said

**Review by R-K-H — APPROVED** [review]

**Pull request merged**